app/middlewares/auth.js

```diff
@@ -12,6 +13,10 @@ exports.checkAuth = (req, _, next) => {
     const verify = validateJwt(token);
     if (!verify.isValid) throw errors.authenticationError(TOKEN_INVALID);
     req.payload = verify.payload;
+    const { id: userId, iat } = verify.payload;
+    const { tokens_expired: tokensExpired } = await userFindByPk(userId);
```

Comment:
If the user doesn't exist, what would happen?

Reply:
If user doesn't exists, an uncontrolled error would be executed. I will change this. 👀 👀 👀